client/src/components/layout/BuildShow.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState, useEffect } from "react"
+
+const BuildShow = (props) => {
+    const [build, setBuild] = useState({})
+
+    const id = props.match.params.id
+
+    const getBuild = async () => {
+        try{
+            const response = await fetch(`/api/v1/builds/${id}`)
+            if(!response.ok) {
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw error
+            }
+            const body = await response.json()
+            setBuild(body.selectedBuild)
+        } catch(error) {
+            console.log(error)
+        }
+    }
+    
+    useEffect(() => {
+        getBuild()
+    }, [])
+
+    return (
+        <div>
```

Comment:
no empty divs! divs must be used for styling

instead, use a react gragment <> </>